docs/source/system_reference_guide/oss_documentation/maap_zenodo.ipynb

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
<p>Can you clarify what "Repositories" refers to? Could be Data or Code depending on users background?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>also be clear, only things developed on MAAP, or using MAAP data will be accepted</p>